app/cli/cmd/policy_develop_lint.go

```diff
@@ -59,7 +59,7 @@ func newPolicyDevelopLintCmd() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&policyPath, "policy", "p", ".", "Path to policy directory")
+	cmd.Flags().StringVarP(&policyPath, "policy", "p", "policy.yaml", "Path to policy file")
```

Comment:
should we do the same approach in eval?

Also, should we change init so the default created file is `policy.yaml`?

Reply:
actually `eval` works like this it seems, it doesn't load a default though. 

the `init` might need update too